mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -113,14 +117,18 @@ def get_signed_conformity_scores(
 
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
+        values: ArrayLike,
```

Comment:
UPD: according to the comments on the `conformity_score.py` file, prefer to use `conformity_scores`.

Reply:
Done